docs/src/development/variables.md

```diff
@@ -73,25 +75,54 @@ If you omit the variable `--value` from the command line as above, you will be p
 
 Changing an environment variable will cause that environment to be redeployed so that it gets the new value.  However, it will *not* redeploy any child environments. If you want those to get the new value you will need to redeploy them yourself.
 
-Environment variables are a good place to store values that apply only on Platform.sh and not on your local development environment. This includes API credentials for 3rd party services, mode settings if your application has a separate "Dev" and "Prod" runtime toggle, etc.
+Environment variables are a good place to store values that apply only on Platform.sh and not on your local development environment. This includes API credentials for 3rd party services, mode settings if your application has a separate "Dev" and "Prod" runtime toggle, etc. 
+
+Build variables (using the `--visible_build` flag) can be a useful place to put those variables which are used to modify your application's build in an environment-dependent way. One example would be to define a Node.js application's build on a production branch using `NODE_ENV=production`, but in development mode (`NODE_ENV=development`) for each of your development environments. Assuming that there is a production, staging, and then a number of development environments that are children of staging, you can do this with the following commands:
+
+```bash
+$ platform variable:create -l environment -e master --prefix env: --name NODE_ENV --value production --visible_build true --inheritable false
+$ platform variable:create -l environment -e staging --prefix env: --name NODE_ENV --value development --visible_build true --inheritable true
+```
+
+With these two commands, while `NODE_ENV` will be `production` on the default branch, it will be `development` on staging as well as each of its child environments. Note that build environment variables change the application's build configuration ID - value updates will trigger a rebuild of the application the same way that a commit would. 
```

Comment:
```suggestion
With these two commands, while `NODE_ENV` will be `production` on the default branch, it will be `development` on staging as well as each of its child environments. Note that build visible environment variables change the application's build configuration ID, and triggers a rebuild of the application in the same way that a commit would.. 
```